### DISCUSSION TITLE
OCTA Support

### DISCUSSION THREAD

**Discussion by AndriasScheuchzeri** [marksgraham/OCT-Converter#25] [General]
Hey Mark, great work!
I am wondering whether your Converter / Reader also works for Topcon - OCTA? 

Stay healthy!

Best, Ansgar

**Comment by marksgraham** [comment]
Hi Ansgar,

Short answer is I don't know! I haven't had the opportunity to try it on OCTA data, but it is possible the data is stored in a similar enough way to the OCT data that the code would work / would only require small modifications. What is the file extension of the Topcon OCTA data? Have you tried just running the OCT code on a file to see what happens?

Best,
Mark
